sdcm/cluster_k8s/mini_k8s.py

```diff
@@ -221,11 +221,16 @@ def get_scylla_cluster_helm_values(self, cpu_limit, memory_limit, pool_name: str
     def create_kubectl_config(self):
         """Kubectl config gets created when K8S cluster is started"""
         LOGGER.info("Creating kubectl config")
-        if self._target_user:
-            self.host_node.remoter.run(self._create_kubectl_config_cmd)
-            # Create config for target user in it's default place
-            self.host_node.remoter.run(f'KUBECONFIG="" {self._create_kubectl_config_cmd}')
-        self.host_node.remoter.sudo(f'bash -cxe " KUBECONFIG=\'\' {self._create_kubectl_config_cmd}"')
+        default_kube_config_dir_path = os.path.expanduser("~/.kube")
+        kube_paths = {self.kube_config_dir_path, default_kube_config_dir_path}
+        for kube_path in kube_paths:
+            if not os.path.exists(kube_path):
+                os.makedirs(kube_path, exist_ok=True)
+            if self._target_user:
+                self.host_node.remoter.sudo(
+                    f'bash -cxe "chown -R {self._target_user}:{self._target_user} {kube_path}"')
+        self.host_node.remoter.run(f'KUBECONFIG="{self.kube_config_path}" {self._create_kubectl_config_cmd}')
```

Comment:
Can you explain what this duplicate is needed ?

Reply:
Line 232 exports kubeconfig to the test-specific dir where we keep test-specific http cache and helm files.
Line 233 exports kubeconfig to the default kube dir where user may have lots of other contexts and which he may use not only for the kind cluster but any other.
So, locally running test will work just fine in parallel to use of `kubectl` to another K8S cluster, for example, EKS from another task.

Reply:
O.k.

It maybe warrants a comment

Reply:
Done